upstream-sha256: 55efac58bb4f22891fb5591918c9e7d70e48d5c557cf9dee859682bb5b17d1c0
actual-sha256: 9327b93806c537762f3925c2b66f7c1825c9dac0c5b940813cecaeb23a12252c
--- upstream.expected
+++ tolk.actual
@@ -24,7 +24,7 @@
   for (int Ridx2 = 0; Ridx2 < tc_k_; Ridx2++) {
     half2 val0 = (*((device half2*)((data0_16384+(alu4+alu0+(Ridx2<<3)+alu1)))));
     half2 val1 = (*((device half2*)((data1_16384+(alu2+alu3+(Ridx2<<10))))));
-    float2 wmma0 = __WMMA_8_8_8_half_float(val0, val1, float2((*(buf0+0)),(*(buf0+1))));
+    float2 wmma0 = __WMMA_8_8_8_half_float(val0, val1, float2((0.0f+(*(buf0+0))),(0.0f+(*(buf0+1)))));
     *(buf0+0) = wmma0.x;
     *(buf0+1) = wmma0.y;
   }
